email/exporter.go

```diff
@@ -116,6 +124,32 @@ func (impl *ExporterImpl) SendContacts(ctx context.Context, req *emailpb.SendCon
 	return &emptypb.Empty{}, nil
 }
 
+// SendCase immediately submits a new Case to the Salesforce REST API using the
+// provided details. Any retries are handled internally by the SalesforceClient.
+func (impl *ExporterImpl) SendCase(ctx context.Context, req *emailpb.SendCaseRequest) (*emptypb.Empty, error) {
+	if core.IsAnyNilOrZero(req, req.Origin) {
+		return nil, berrors.InternalServerError("Incomplete gRPC request message")
+	}
+
+	err := impl.client.SendCase(Case{
+		Origin:        req.Origin,
+		Subject:       req.Subject,
+		Description:   req.Description,
+		ContactEmail:  req.ContactEmail,
+		Organization:  req.Organization,
+		AccountId:     req.AccountId,
+		RateLimitName: req.RateLimitName,
+		RateLimitTier: req.RateLimitTier,
+		UseCase:       req.UseCase,
+	})
+	if err != nil {
+		impl.caseErrorCounter.Inc()
+		return nil, berrors.InternalServerError("Sending Case to the Salesforce REST API: %s", err)
```

Comment:
style: error messages shouldn't start with capital letters. We do capital letters in the WFE when creating the outermost message that will be directly surfaced to the user (i.e. in sendError), but try to avoid them everywhere else.